app/main.py

```diff
@@ -2,0 +1,7 @@
+def copy_file(command: str) -> None:
+    command_list = command.split()
+    if len(command_list) == 3 and command_list[1] != command_list[2]\
+            and command_list[0] == "cp":
+        with (open(command_list[1], "r") as main_file,
+              open(command_list[2], "w") as file_copy):
+            file_copy.write(main_file.read())
```

Comment:
The way you open and write to files in python is correct, but it's not the best practice. It's better to open files using the `with` statement one by one. This way, if an error occurs while opening the second file, the first file will still be closed properly. Here is a better way to do it:

python
with open(command_list[1], 'r') as main_file:
    data = main_file.read()
with open(command_list[2], 'w') as file_copy:
    file_copy.write(data)
```